spark-3/src/main/java/com/google/cloud/spark/bigquery/spark3/DataFrameToRDDConverter.java

```diff
@@ -0,0 +1,10 @@
+package com.google.cloud.spark.bigquery.spark3;
```

Comment:
Minor and maybe not relevant: maybe it makes sense to move that interface under bigquery.connector.common - as it defines behaviour of not only spark3.

Reply:
Thanks for the comment - `bigquery.connector.common` is spark agnostic, hence this code cannot reside there.